boards/xtensa/m5stickc_plus/m5stickc_plus.dts

```diff
@@ -5,7 +5,7 @@
  */
 /dts-v1/;
 
-#include <espressif/esp32/esp32_wrover_e_n4r8.dtsi>
+#include <espressif/esp32/esp32_pico_d4.dtsi>
```

Comment:
Is it a fixup of previous commit? Which SiP is the right one?

Reply:
The last one is correct.